src/green_recognition/include/CircleDetection.h

```diff
@@ -0,0 +1,108 @@
+/*
+ * Created By: Robyn Castro
+ * Created On: June 15, 2017
+ * Description: Determines whether or not green is seen on
+ *              the screen then publishes a recommended twist
+ *              message.
+ * Usage:
+ * Subscribes to: camera // TODO: Determine topic names
+ * Publishes to:
+ */
+
+#ifndef GREEN_RECOGNITION_H
+#define GREEN_RECOGNITION_H
+
+// OpenCV
+#include <opencv2/imgproc/imgproc.hpp>
+#include <opencv2/core/core.hpp>
+#include <opencv2/opencv.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/objdetect/objdetect.hpp>
+
+// Image Conversion
+#include <cv_bridge/cv_bridge.h>
+#include <sensor_msgs/image_encodings.h>
+#include <image_transport/subscriber.h>
+
+// ROS
+#include <ros/ros.h>
+#include <ros/ros.h>
+#include <geometry_msgs/Twist.h>
+
+// STD
+#include <vector>
+#include <string>
+
+// Snowbots
+#include <sb_utils.h>
+
+using namespace cv;
+
+class GreenRecognition {
```

Comment:
Is `GreenRecognition` descriptive of what this node is? Something like `GreenLightDetection` might be a bit better. Or better yet, just `TrafficLightDetection`, as we may want to generalise this to recognise lights of any color.